src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +288,26 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                if (
```

Comment:
We need a test block for this. Since this is not live yet you can follow up after this PR if it makes things easier:
<img width="639" alt="image" src="https://user-images.githubusercontent.com/44410/212391418-ef12d53a-16f5-46c8-bd8c-41f44ffcc42a.png">